site/docs/master/sonobuoy-config.md

```diff
@@ -54,7 +54,14 @@ Filters
    - A Kubernetes [label selector][labelselector] which will be added to every query run.
 
 Limits
- - Options for limiting the size of the pod logs (in bytes) or the how far back in time to gather logs (in seconds). These will be passed onto Kubernetes [PodLogOptions][podlogopts]
-
+ - Options for limiting limits the scope of response when getting logs from pods. These will be passed onto Kubernetes [PodLogOptions][podlogopts]
```

Comment:
💯 For updating the docs :)

Can you also update https://github.com/heptio/sonobuoy/blob/master/site/docs/master/sonobuoy-config.md#query-options ?

Reply:
Oh I misread; then you were spot on. I was thinking you were updating the master readme, but you accurately updated the master version of this doc. 👍 Sorry for any confusion I made.